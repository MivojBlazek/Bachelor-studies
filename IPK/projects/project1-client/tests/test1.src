/auth username Password DisplayName
/join channel1
Hello, this is a message to the current channel.
