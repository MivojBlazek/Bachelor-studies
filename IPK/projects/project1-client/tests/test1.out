Success: Auth success.
Server: Display Name has joined default.
Success: Join success.
Server: Display Name has joined channel1.